tests/lib/snaps/store/test-snapd-mount-control-nfs/bin/cmd

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+PS1='$ '
+command="$1"
+shift
+
+exec "$command" "$@"
```

Comment:
Personally I much prefer the .sh style of tests but you don't have to change it.